src/System.Private.CoreLib/src/System/Reflection/Emit/ILGenerator.cs

```diff
@@ -195,12 +172,17 @@ internal void UpdateStackSize(OpCode opcode, int stackchange)
             // each basic block.
             if (opcode.EndsUncondJmpBlk())
             {
-                m_maxStackSize += m_maxMidStack;
-                m_maxMidStack = 0;
-                m_maxMidStackCur = 0;
+                FixupBasicBlock();
```

Comment:
Does refactoring this out into a separate method reduce the native code size in any meaningful way? It looks unlikely to me.

Reply:
It did, but very minor differences, I removed it